test_plone43.cfg

```diff
@@ -6,8 +6,8 @@ extends =
     base.cfg
 
 [versions]
-zc.buildout = 2.13.1
-setuptools = 40.8.0
+# Keep sync with requirements.txt to avoid infinite loop in buildout
+setuptools = 42.0.2
+zc.buildout = 2.13.3
 
-configparser = 4.0.2
```

Comment:
I am not sure why this was unpinned

Reply:
It is already pinned on qa.cfg:

https://github.com/collective/buildout.plonetest/blob/e402268312566282a546ee9d7a8137307d2feed5/qa.cfg#L130